kDriveCore/Data/Models/Drive/Drive.swift

```diff
@@ -60,13 +60,13 @@ public enum MaintenanceReason: String, PersistableEnum, Codable {
     case technical
 }
 
-public class DrivePreferences: EmbeddedObject, Codable {
+public final class DrivePreferences: EmbeddedObject, Codable {
     @Persisted public var color = "#0098FF"
     @Persisted public var hide = false
 }
 
-public class Drive: Object, Codable {
-    @Persisted(primaryKey: true) public var objectId = ""
+public final class Drive: Object, Codable {
+    @Persisted(primaryKey: true) public var objectId: String = UUID().uuidString
```

Comment:
This is not correct, objectId != UUID().uuidString

Reply:
Is it less correct than the previous default that was empty string ? `""`

Reply:
I'll change this to only set a UUID in the empty init method.

Reply:
I wanted to ensure that we don't have places in the code where we check for .isEmpty on the objectId but this is not the case. So everything seems good 👍